graphql-dgs/src/test/kotlin/com/netflix/graphql/dgs/internal/InputArgumentTest.kt

```diff
@@ -1436,39 +1455,47 @@ internal class InputArgumentTest {
 
         val fetcher = object : Any() {
             @DgsData(parentType = "Query", field = "hello")
-            fun someFetcher(@InputArgument someInput: InputMessage): String {
-                return "Hello, this is a ${someInput.type} greeting"
+            fun hello(@InputArgument input: KInputMessage): String {
```

Comment:
In the past we were just asserting the representation of the output as a string. We need to assert that the collections actually contain the proper type.